apps/etterlatte-brev-api/src/test/kotlin/no/nav/etterlatte/brev/behandling/SakOgBehandlingServiceTest.kt

```diff
@@ -0,0 +1,205 @@
+import no.nav.etterlatte.libs.common.soeknad.dataklasser.common.PersonType
+import no.nav.etterlatte.libs.common.soeknad.dataklasser.common.Spraak
+import no.nav.etterlatte.libs.common.toJsonNode
+import no.nav.etterlatte.libs.common.vedtak.Attestasjon
+import no.nav.etterlatte.libs.common.vedtak.Periode
+import no.nav.etterlatte.libs.common.vedtak.Vedtak
+import no.nav.etterlatte.libs.common.vedtak.VedtakFattet
+import no.nav.etterlatte.libs.common.vedtak.VedtakType
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.time.Instant
+import java.time.LocalDate
+import java.time.YearMonth
+import java.time.ZonedDateTime
+import java.util.*
+
+internal class SakOgBehandlingServiceTest {
+
+    private val vedtaksvurderingKlient = mockk<VedtaksvurderingKlient>()
+    private val grunnlagKlient = mockk<GrunnlagKlient>()
+    private val beregningKlient = mockk<BeregningKlient>()
+    private val saksbehandlere = mapOf(
+        SAKSBEHANDLER_IDENT to "0904"
+    )
+
+    private val service =
+        SakOgBehandlingService(vedtaksvurderingKlient, grunnlagKlient, beregningKlient, saksbehandlere)
+
+    @BeforeEach
+    fun before() {
+        clearAllMocks()
+    }
+
+    @AfterEach
+    fun after() {
+        confirmVerified(vedtaksvurderingKlient, grunnlagKlient, beregningKlient)
+    }
+
+    @Test
+    fun `SakOgBehandling fungerer som forventet`() {
+        coEvery { vedtaksvurderingKlient.hentVedtak(any(), any()) } returns opprettVedtak()
+        coEvery { grunnlagKlient.hentGrunnlag(SAK_ID, ACCESS_TOKEN) } returns opprettGrunnlag()
+        coEvery { beregningKlient.hentBeregning(any(), any()) } returns opprettEnkelBeregning()
+
+        val behandling = runBlocking {
+            service.hentBehandling(SAK_ID, BEHANDLING_ID, SAKSBEHANDLER_IDENT, ACCESS_TOKEN)
+        }
+
+        assertEquals(SAK_ID, behandling.sakId)
+        assertEquals(BEHANDLING_ID, behandling.behandlingId)
+        assertEquals(Spraak.NB, behandling.spraak)
+        assertEquals("Innsend Innsender", behandling.persongalleri.innsender.navn)
+        assertEquals("Soek Soekeren", behandling.persongalleri.soeker.navn)
+        assertEquals("Avdoed Avdoederen", behandling.persongalleri.avdoed.navn)
+        assertEquals(VedtakType.INNVILGELSE, behandling.vedtak.type)
+        assertEquals(123L, behandling.vedtak.id)
+        assertEquals(SAKSBEHANDLER_IDENT, behandling.vedtak.saksbehandler.ident)
+        assertEquals(ATTESTANT_IDENT, behandling.vedtak.attestant?.ident)
+        assertEquals(YearMonth.now().atDay(1), behandling.utbetalingsinfo?.virkningsdato)
+
+        coVerify(exactly = 1) { vedtaksvurderingKlient.hentVedtak(BEHANDLING_ID, ACCESS_TOKEN) }
+        coVerify(exactly = 1) { grunnlagKlient.hentGrunnlag(SAK_ID, ACCESS_TOKEN) }
+        coVerify(exactly = 1) { beregningKlient.hentBeregning(BEHANDLING_ID, ACCESS_TOKEN) }
```

Comment:
på disse er det egentlig litt nice å bruke `any()` i steden for de faktiske verdiene, for da dekker de casene om at man ikke skal gjøre requests med andre argumenter også 🤠

Reply:
Tenkte det er greit å få sjekket at man kaller videre med riktig Sak og behandlingsid. Er også slik vi har gjort det i de andre testene i brev repoet. 
For access token kan vi dog bytte til any()